src/lib/core/style/_noop-animation.scss

```diff
@@ -0,0 +1,10 @@
+// If the mat-animation-noop class is present on the components root element,
+// prevent non css animations from running.
+// NOTE: Currently this mixin should only be used with components that do not
+// have any projected content.
+@mixin _noop-animation() {
```

Comment:
`_mat-noop-animation`

Reply:
can we call it `noopable`? it's not nooping the animation, it's just making it noopable in case you add the noop class